recipe/build.sh

```diff
@@ -1,15 +1,30 @@
 #!/bin/sh
 
-libtoolize --copy
-autoreconf -vfi
-
-./configure          \
-  --prefix=${PREFIX} \
-  --enable-static    \
-  --enable-shared    \
-  --without-octave   \
-  --without-matlab   \
-  --without-guile
-
-make -j${CPU_COUNT} ${VERBOSE_AT}
-make install ${VERBOSE_AT}
+set -euo pipefail
+
+if [ "$(uname)" = "Darwin" ]; then
+  # osx uses a temp directory which messes up with cmake finding libraries.
+  cd "${SRC_DIR}" || exit
+fi
+
+mkdir build && cd build || exit
```

Comment:
😆 the point about `set -e` it that `cd`, `mkdir` etc. will cause the script to exit if they fail.  You don't need `|| exit [1]`

```suggestion
  cd "${SRC_DIR}"
fi

mkdir build
cd build
```

Also, putting one command per line (`mkdir` ... `cd`) makes a script more readable for the nervous -- and you've captured the "logical and" (`&&`) with `set -e`, above.